src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { Loader } from '../Loader';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { completed, title } = todo;
+  const [isBeingEdited] = useState(false);
```

Comment:
The isBeingEdited state should have a setter function, e.g., const [isBeingEdited, setIsBeingEdited] = useState(false);